wiki/Mapping_and_Modding_Timeline/en.md

```diff
@@ -0,0 +1,458 @@
+
+### August
+
+- **2016-08-05:** The [Quality Assurance Team blog](/wiki/Quality_Assurance_Team_Blog) was created.
+    - The QAT was receiving criticism about being too closed-off from the rest of the mapping community, so this blog was created to increase transparency for QAT decision making and community activity. <!-- https://osu.ppy.sh/community/forums/topics/578437 -->
+- **2016-08-26:** The [Code of Conduct](/wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping) was implemented to establish modding behavior expectations.
+    - These rules and guidelines were treated as common sense up until this point. By documenting them, moderation could be more consistent.<!-- https://osu.ppy.sh/community/forums/topics/447557 -->
+
+### September
+
+- **2016-09-30:** The [Community Mentorship Program](/wiki/Community_Mentorship_Program) re-launched after a long hiatus.
+    - Its new structure was centered around [Discord](https://discord.com/new) communication and seasonal cycles.
+    - The program was led by [Okoratu](https://osu.ppy.sh/users/1623405) and [Yuii-](https://osu.ppy.sh/users/2935923) <!-- https://osu.ppy.sh/community/forums/topics/506906 -->
+
+### October
+
+- **2016-10-17:** [Loved](/wiki/Beatmaps#loved) beatmap category was created.
+    - The category provided leaderboards for popular maps that could not be ranked for various reasons.
+    - *See [History of Loved](/wiki/Beatmaps/History_of_Loved) for a detailed timeline of Loved system changes.*
+
+### November
+
+- **2016-11-01:** The [Beatmap Veto](/wiki/People/The_Team/Beatmap_Nominators/Beatmap_Veto) system was implemented.
+    - Beatmap Nominators could halt a beatmap in the ranking process for subjective reasons by resetting a nomination. This was done with a `bubble pop` (![bubble pop icon](/wiki/shared/icon/bubble-pop.gif)) icon.
+    - Vetoes could be overwritten by the nominations of Beatmap Nominators previously uninvolved with the beatmap.
+    - To allow adequate time to veto, consecutive nominations could not be set within 24 hours of each other.
+    - Inconclusive vetoes were mediated by the QAT shortly after this system's implementation. <!-- date is not exact at all, but logs are from deleted discord chat -->
+- **2016-11-02:** Beatmap Nominator files were created to keep tabs on all individual BN members.
+    - These files included modding/nomination activity, disqualifications, and behavior issues.
+    - Members of the BN with 3 or more "strikes" logged in these files would be removed from the group. <!-- QAT folder archives -->
+
+**1720 beatmaps** were ranked in 2016.
+
+## 2017
+
+### April
+
+- **2017-04-06:** [Beatmap Nominator Tiers](/wiki/Modding/Beatmap_Nominator_Tiers) were implemented, splitting the BN into two subgroups called "tiers".
+    - Because the QAT was no longer acting as a quality control gate, this system was implemented to improve the quality of beatmaps entering ranked.
```

Comment:
```suggestion
    - Because the QAT was no longer acting as a quality control gate, this system was implemented to improve the quality of beatmaps entering Ranked.
```